docs/source/installation.md

```diff
@@ -14,7 +14,7 @@ Before getting started, make sure you have the following tools installed:
 
 To install Sphinx and this theme in a BlueRobotics project, follow these steps:
 
-1. Copy the `docs` folder [from this repository](https://github.com/bluerobotics/sphinx-blue-robotics-theme) into your own project.
+1. Copy the `docs` and `.github` folders [from this repository](https://github.com/bluerobotics/docs-boilerplate) into your own project.
```

Comment:
@patrickelectric Could you create a new repo with a name like this (or something similar)? We'll set up a docs project with minimal configuration to serve as a boilerplate for other projects looking to install the theme.  

In the future, you can also use it for uploading a style guide or documentation templates to ensure consistency (e.g. how to write guides, mandatory pages or sections all projects must have, ...).

Reply:
@dgarcia360 done!
https://github.com/bluerobotics/docs-boilerplate